grr/server/grr_response_server/gui/ui/lib/api/http_api_service.ts

```diff
@@ -234,14 +245,14 @@ export class HttpApiService {
 
     const params = new HttpParams({
       fromObject: {
-        start: ((start?.getTime() ?? 1) * 1000).toString(),  // If not set, fetch from beggining of time
+        start: ((start?.getTime() ?? 1) * 1000)
+                   .toString(),  // If not set, fetch from beggining of time
```

Comment:
I am not sure if this is prettier than the previous version. Cannot we just put the comment about the line?